scene/resources/tile_set.cpp

```diff
@@ -4716,14 +4716,19 @@ void TileSetScenesCollectionSource::set_scene_tile_id(int p_id, int p_new_id) {
 void TileSetScenesCollectionSource::set_scene_tile_scene(int p_id, Ref<PackedScene> p_packed_scene) {
 	ERR_FAIL_COND(!scenes.has(p_id));
 	if (p_packed_scene.is_valid()) {
-		// Make sure we have a root node. Supposed to be at 0 index because find_node_by_path() does not seem to work.
-		ERR_FAIL_COND(!p_packed_scene->get_state().is_valid());
-		ERR_FAIL_COND(p_packed_scene->get_state()->get_node_count() < 1);
-
 		// Check if it extends CanvasItem.
-		String type = p_packed_scene->get_state()->get_node_type(0);
+		Ref<SceneState> scene_state = p_packed_scene->get_state();
+		String type = "";
```

Comment:
`String` is default allocated so the `= ""` is not needed.

Reply:
Fixed, thanks!